src/proc.h

```diff
@@ -40,4 +42,7 @@ s_error* h_proc_export_csv(s_sheet* sheet, const char* path);
 /** Generate a filename from a given month and an extension */
 char* h_proc_generate_filename(e_month month, const char* extension);
 
+/** Comment */
+s_error* h_processing(s_sheet* sheet, s_arr_irs irs_array, s_arr_seg_social ss_array);
```

Comment:
Outra cena, o utilizador pode entretando ter apagado toda a tabela tanto de IRS como da Seg Social, sendo assim nas duas a variavel `used` deve estar a 0 e `data` deve estar a `NULL` se for o caso retorna `s_error` a dizer que não é possivel processar o salario!